specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -335,8 +335,7 @@ If a container supports Java SE 11 or a newer version of the Java SE platform, t
 
 During the development cycle for the current version of the Jakarta EE specification, the platform project considered several component specifications for inclusion in the platform. A consensus could not be reached on including these specifications in the platform. These specifications are considered as prospects for inclusion in a future version of the Platform specification.
 
-* [Jakarta MVC](https://jakarta.ee/specifications/mvc/)
-* [Jakarta NoSQL](https://jakarta.ee/specifications/nosql/)
```

Comment:
Why was this line removed?

Reply:
One of the requirements for being listed as a Prospective spec is to have released at least one final version of the specification. NoSQL is still a beta.
The can be added back if they release a final version before EE 11.